content/articles/deploy-django-with-heroku/index.md

```diff
@@ -0,0 +1,478 @@
+![hero](/how-to-deploy-django-project-with-database-using-heroku-cloud-hoisting/django-heroku.png)
+
+### Introduction 
+
+Welcome folks! I am Arafat Olayiwola, a software developer and technical writer. In this tutorial I will be walking you through steps of deploying Django application with relational database using Heroku server by building a `studentprofile` application. Heroku is one of the cloud providers out there that allows developers to hoist their application on their server, and helps by monitoring, managing the health checks of the server. 
+
+To deploy Django application on Heroku server, some dependencies are needed to set up and we shall do justice to that in this article. Feel free to brush up concepts of Django that require mastering in this article [Core Concepts of Django to master](/https://section.io/engineering-education/core-concepts-of-django-to-learn/), though it is not a prequisite for this.
```

Comment:
1. Don't include your name here, that will be added by the peer reviewer.
2. User clear simple and concise sentences. Each sentence should make meaning on its own.
3. Communicate a single idea per paragraph.